iothub/device/src/IotHubBaseClient.cs

```diff
@@ -60,6 +60,7 @@ public abstract class IotHubBaseClient : IAsyncDisposable
                 ModelId = _clientOptions.ModelId,
                 PayloadConvention = _clientOptions.PayloadConvention,
                 IotHubClientTransportSettings = _clientOptions.TransportSettings,
+                FileUploadTransportSettings = _clientOptions.FileUploadTransportSettings,
```

Comment:
This is in preparation for adding file upload into the delegating handler pipeline